docs/src/Typeahead.doc.js

```diff
@@ -26,7 +26,7 @@ card(
         href: 'basicExample',
       },
       {
-        name: 'defaultItem',
```

Comment:
@simkessy this would be a semver major change. If we do want to change the name, could you add a codemod so we (and others) can easily upgrade gestalt in a repo?

A good example is https://github.com/pinterest/gestalt/blob/master/packages/gestalt-codemods/1.8.0-1.9.0/heading-size-replace.js